src/app/loaders/express/locals.ts

```diff
@@ -64,6 +68,9 @@ const environment = ejs.render(
     var reactMigrationRespondentRolloutStorage = "<%= reactMigrationRespondentRolloutStorage%>"
     var reactMigrationAdminRollout = "<%= reactMigrationAdminRollout%>"
     var reactMigrationAngularPhaseOutDate = "<%= reactMigrationAngularPhaseOutDate%>"
+    // Payment
+    var maxPaymentAmountCents = "<%= maxPaymentAmountCents%>"
+    var minPaymentAmountCents = "<%= minPaymentAmountCents%>"
```

Comment:
There is no need to convert the values from number to string. These are are env vars where we control the type in the config definition

```suggestion
    var maxPaymentAmountCents = <%= maxPaymentAmountCents%>
    var minPaymentAmountCents = <%= minPaymentAmountCents%>
```

Side note not related to the PR: I'm not sure why we supply the env vars as old-style `var` defined JS, rather than using JSON (which is also typed). We could probably revisit and simplify this at some point.